e2e/outbound_e2e_test.go

```diff
@@ -40,15 +38,19 @@ func TestE2EOutboundRelay(t *testing.T) {
 	resp, err := outboundRelayClient.Post(syntheticEndpointUrl, "application/json", bytes.NewReader(payload))
 	require.NoError(t, err)
 
-	body, err := ioutil.ReadAll(resp.Body)
+	body, err := io.ReadAll(resp.Body)
 	require.NoError(t, err)
 
 	// close response body
-	resp.Body.Close()
+	err = resp.Body.Close()
+	if err != nil {
+		log.Printf("Failed to close response body: %s", err)
+	}
 
 	responseData := make(map[string]map[string]bool)
 
-	json.Unmarshal(body, &responseData)
+	//nolint:errcheck
+	_ = json.Unmarshal(body, &responseData)
```

Comment:
No point in ignoring errors here 

```suggestion
	err = json.Unmarshal(body, &responseData)
	require.NoError(t, err)
```